src/apps/front-office/home/hooks/use-fetch-deals.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from "react";
+import { DealsData, getDeals } from "../services/home-service";
+
+export default function useFetchDeals() {
+  const [data1, setData1] = useState<DealsData[] | []>([]);
+  const [_loading, set_Loading] = useState(true);
```

Comment:
We don't use `snake_case` syntax, please use `camelCase` instead to define the the `setLoading` state update